Create 30-03-2021.mdx (#111)
meeshkan/website: src/changelog/30-03-2021.mdx

---
version: "0.5.6"
date: 30-03-2021
title: Release v0.5.6 | Meeshkan
description: Release v0.5.6 | Keep up to date on the development of the Meeshkan product.
slug: 30-03-2021
authors: ["ben"]
published: true
tags:
  - changelog
---

# Test Runner in stealth mode 🥷
You can already run tests you create from User Stories by [exporting Test Cases as Puppeteer scripts](https://meeshkan.com/blog/how-to-puppeteer-jest-github-action/) but we're also working to allow you to run them directly from within Meeshkan, and we've made some big strides in bringing that to life this cycle. <a onClick={() => {window.Intercom("showMessages")}}>Get in touch</a> to get early access!

> You can now clearly distinguish the **Log-in flow, authenticated flows, and unauthenticated flows** too, with handy tags in an individual user story. **All user stories have videos generated automatically**, and playable directly from the _Recordings_ table.

Author: ```suggestion

```

> You can now clearly distinguish the **Log-in flow, authenticated flows, and unauthenticated flows** too, with handy tags in an individual user story. **All user stories have videos generated automatically**, and playable directly from the _Recordings_ table.

Author: ```suggestion
You can now also clearly distinguish the **log-in flow, authenticated flows,** and **unauthenticated flows** too, with handy tags that are automatically added to the appropriate user story. Videos are also now **generated automatically for all User Stories**, and playable directly from the _Recordings_ table.
```
